lib/modules/manager/maven/extract.ts

```diff
@@ -39,6 +39,27 @@ function parsePom(raw: string, packageFile: string): XmlDocument | null {
   return null;
 }
 
+function parseExtensions(raw: string, packageFile: string): XmlDocument | null {
+  let extensions: XmlDocument;
+  try {
+    extensions = new XmlDocument(raw);
+  } catch (err) {
+    logger.debug({ packageFile }, `Failed to parse as XML`);
+    return null;
+  }
+  const { name, attr, children } = extensions;
+  if (name !== 'extensions') {
+    return null;
+  }
+  if (attr.xmlns === 'http://maven.apache.org/EXTENSIONS/1.0.0') {
+    return extensions;
+  }
```

Comment:
it looks like this should be inverted, otherwise you never check for children length on extensions